VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseCardViewHolder.java

```diff
@@ -52,7 +52,7 @@ public void setCourseTitle(@NonNull String title) {
 
     public void setCourseImage(@NonNull String imageUrl) {
         Glide.with(courseImage.getContext())
-                .load((String)null)
```

Comment:
It was the same as the outcome of this change. Snuck in when I was debugging something and didn't mean to commit it